src/control/lib/control/pool.go

```diff
@@ -625,7 +625,7 @@ func PoolQuery(ctx context.Context, rpcClient UnaryInvoker, req *PoolQueryReq) (
 	return pqr, err
 }
 
-// Update the pool state
+// UpdateState update the pool state.
```

Comment:
spelling nit
```suggestion
// UpdateState updates the pool state.
```

Reply:
Thank you @kjacque, I will update this in next PR.